common/src/main/java/org/jboss/gm/common/Configuration.java

```diff
@@ -171,6 +174,18 @@ default Properties getProperties() {
     @DefaultValue("")
     String manipulationVersion();
 
+    @Key("reportTxtOutputFile")
+    @DefaultValue(REPORT_TXT_OUTPUT_FILE)
```

Comment:
Why do we need the field here? Why not just `""` ?

Reply:
I originally had `""`. I changed it

1. To be consistent with the JSON property.
2. I thought I might have to reference it to set the value from within the tests (but I guess not).
3. It could make sense to make all the keys constants. Otherwise, you have to remember what the key is called and possibly get an error whereas if it was a public constant the IDE would just look it up and ensure you do not mistype or retype these.

If I look at PME both the key and value are constants, which I like:

```
final String reportJsonOutputFile = session.getUserProperties().getProperty( REPORT_JSON_OUTPUT_FILE,
                                                                                             session.getTargetDir() + File.separator + REPORT_JSON_DEFAULT);
``

But, you see that here it is able to get the `targetDir` on the fly, we can't do this because the default is set ahead of time using the `@DefaultValue`, right?

Reply:
I am not seeing the need right now to make all the default values as separate constants. I think making this `""` is sufficient for now.

Reply:
Also, with the keys, maybe it's not necessary if you have access to the configuration object, but I see tests setting properties and they need to know the name of the properties to set.

Reply:
Right but we're not talking about keys here but the default value

Reply:
OK, I left the key constant to refer to in the test, and removed the other, just using hardcoded `""`.